src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -60,6 +88,14 @@ private void OnOptionChanged(object sender, OptionChangedEventArgs e)
                 AddOrRemoveDropdown(enabled);
             }
 
+            private WorkspaceRegistration GetWorkspaceRegistration()
```

Comment:
Is this ever called anywhere other than the constructor? Just easier to inline in that case?